tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot

```diff
@@ -48,6 +48,19 @@ Create VCH - target URL
     Run Regression Tests
     Cleanup VIC Appliance On Test Server
 
+Create VCH - operations user
+    Set Test Environment Variables
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} --bridge-network=%{BRIDGE_NETWORK} --external-network=%{EXTERNAL_NETWORK} ${vicmachinetls} --ops-user="%{TEST_USERNAME}" --ops-password="%{TEST_PASSWORD}"
```

Comment:
ideally, but we'd then need to set that up on all test systems as well, expose it throughout the CI logic.
It's absolutely required once we start configuring the RBAC roles, but not just for allowing distinction between administrative user for deployment and operations user for function.